rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -455,11 +455,25 @@ extern void ppu_register_function_at(u32 addr, u32 size, ppu_function_t ptr)
 	}
 }
 
+extern atomic_t<bool> g_debugger_pause_all_threads_on_bp;
```

Comment:
Almost forgot. It should be initialized here because it belongs to emucore, instead of demanding external initialization.